.github/workflows/test.yml

```diff
@@ -37,4 +37,9 @@ jobs:
       uses: docker-practice/actions-setup-docker@0.0.1
     - name: Tests
       run: |
-        poetry run python -m pytest
+        poetry run python -m pytest --cov-branch --cov=./yellowbox --cov-report=xml tests/
```

Comment:
are we sure we want branch coverage?

Reply:
Yeah, I usually do it to cover all scenarios even if it reduces the %